src/interfaces/IMetaMorpho.sol

```diff
@@ -30,57 +30,148 @@ interface IOwnable {
 /// @dev This interface is used for factorizing IMetaMorphoStaticTyping and IMetaMorpho.
 /// @dev Consider using the IMetaMorpho interface instead of this one.
 interface IMetaMorphoBase {
+    /// @notice The address of the Morpho contract.
     function MORPHO() external view returns (IMorpho);
 
+    /// @notice The address of the curator.
     function curator() external view returns (address);
+
+    /// @notice Stores whether an address is an allocator or not.
     function isAllocator(address target) external view returns (bool);
+
+    /// @notice The current guardian. Can be set even without the timelock set.
     function guardian() external view returns (address);
 
+    /// @notice The current fee.
     function fee() external view returns (uint96);
+
+    /// @notice The fee recipient.
     function feeRecipient() external view returns (address);
+
+    /// @notice The skim recipient.
     function skimRecipient() external view returns (address);
+
+    /// @notice The current timelock.
     function timelock() external view returns (uint256);
+
+    /// @dev Stores the order of markets on which liquidity is supplied upon deposit.
+    /// @dev Can contain any market. A market is skipped as soon as its supply cap is reached.
     function supplyQueue(uint256) external view returns (Id);
```

Comment:
Does not have `@notice`, `@param` and `@return`

I would have anyway suggested documenting it properly, but I think that in this case is even more important given that the name of the function seems more "active" (by executing you supply to the queue) while instead it's a getter.

Reply:
The `supplyQueue`, as far as I remember, can contain duplicates (I don't think you have changed such behavior).

In the `withdrawQueue` you specify that it does not contain duplicates. You should also specify with a `@dev` comment that `supplyQueue` **can** contain duplicates.